### PR TITLE
Fix #2093: [Mobile-Landscape] High-fi StoryActivity Updated UI

### LINKED ISSUE #2093: [Mobile-Landscape] High-fi StoryActivity Updated UI
The `StoryActivity`  UI has been updated a lot which is available here. https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/cce938d6-e079-4761-bac0-8bde22ac6348/

Major changes:
- [ ] The inactive card content formation is updated.
- [ ] RTL Supportive

**Note : Please test UI for RTL support, prior to merging this fix.**

**Follow this guide**: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by ReCodee**
## Explanation

 Fixes #2093 

### Screenshots
Screenshot_2020-12-01-16-13-31-391_org oppia android

Screenshot_2020-12-01-16-12-36-344_org oppia android

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
You can check and setup ktlint on your local machine, this will prevent the GitHub check failures. 
https://github.com/oppia/oppia-android/wiki/Ktlint-Guide

**Comment by anandwana001** [comment]
@ReCodee I suggest to setup Macros, this is fix all issue in just one shortcut. 
Please check here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

File: app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt [hunk 1]
```diff
@@ -142,6 +146,49 @@ class StoryFragmentPresenter @Inject constructor(
...
+                    view.context
+                      .getText(R.string.chapter_name)
+                  ),
+                  HtmlCompat.TO_HTML_PARAGRAPH_LINES_CONSECUTIVE
+                ),
+                storyItemViewModel.index + 1,
+                storyItemViewModel.name
+              )
+            }
+          binding.storyChapterCard.strokeWidth = when (
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt]:
```kotlin
 binding.storyChapterCard.strokeWidth =
            when (storyItemViewModel.chapterSummary.chapterPlayState) {
              ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ->
                view.context.resources.getDimension(R.dimen.space_0dp).toInt()
              else ->
                view.context.resources.getDimension(R.dimen.story_chapter_stroke_width).toInt()
            }
```

File: app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt [hunk 2]
```diff
@@ -142,6 +146,49 @@ class StoryFragmentPresenter @Inject constructor(
...
+            else ->
+              view.context.resources.getDimension(R.dimen.story_chapter_stroke_width).toInt()
+          }
+          binding.titleHtmlContent =
+            htmlParserFactory.create(
+              resourceBucketName,
+              entityType,
+              storyItemViewModel.storyId,
+              imageCenterAlign = true
+            ).parseOppiaHtml(
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt:188]:
```kotlin
).parseOppiaHtml(
    str,
    binding.chapterTitle
)
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Formating suggestion, @ReCodee

File: app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt]: You are getting the indentation issue. Please use the Reformat Code or Macros as per the suggestion to fix the issue. 

```kotlin
binding.storyChapterCard.strokeWidth =
  when (storyItemViewModel.chapterSummary.chapterPlayState) {
    ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ->
      view.context.resources.getDimension(R.dimen.space_0dp).toInt()
    else ->
      view.context.resources.getDimension(R.dimen.story_chapter_stroke_width).toInt()
  }
```

**Comment by anandwana001** [comment]
@ReCodee Any update on this?

**Comment by ReCodee** [comment]
Will reopen this when ci check fails are resolved.

**Pull request closed without merging**